nmrfx-structure/src/main/java/org/nmrfx/structure/chemistry/predict/HosePrediction.java

```diff
@@ -609,18 +609,11 @@ void validate(String validate) {
                         deltaSums[i] += (ppm - predPPM) * (ppm - predPPM);
                         System.out.printf("%s\t%8.2f\t%8.2f\t%8.2f\t%d\t%d\t%s\t%s\t%s\n", statNames[i], ppm, predPPM, Math.abs(ppm - predPPM), iShell, stat.nValues, hoseString[0], smile, molNum);
                     }
-                    if ((stat == null) || (stat.nValues < 4)) {
-                        if (stat == null) {
-                            //System.err.println(lineReader.getLineNumber() + " " + statNames[i] + " " + hoseString[0] + " null " + iShell + " " +  ppm);
-                        } else {
-                            //System.err.println(lineReader.getLineNumber() + " " + statNames[i] + " " + hoseString[0] + " insufficient " + stat.nValues + " " +  iShell + " " + ppm);
-                        }
-                    } else {
+                    if (!((stat == null) || (stat.nValues < 4))) {
```

Comment:
`x != y` is easier to read that `!(x == y)` 😄  

It also looks like you inverted the logic here.

```suggestion
                    if ((stat != null && stat.nValues >= 4)) {
```

Reply:
Oops! Changed to your suggestion.